### DISCUSSION TITLE
"Data Source Changes Detected" notifications persist even after refresh

### DISCUSSION THREAD

**Discussion by sblack4** [code4recovery/12-step-meeting-list#1599] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/018f941c-ebe5-4be3-b7c1-a0c850158a27)

For this (woodbridge VA) website we have two meetings that continue to show up in the "Data Source Changes Detected" emails even after the data source has been refreshed
- https://woodbridgeaa.org/meetings/?meeting=real-alcoholic-womens-big-book
- https://woodbridgeaa.org/meetings/?meeting=winners-circle

I've tried two methods to fix this 
- Refreshing the data source
- Deleting the meetings and then refreshing the data source

This is also an issue for meetings from several data sources on https://aavirginia.org/meetings/. I can provide the list of meetings there if you need it :)

**Comment by ginslo** [comment]
Hi @sblack4. Can you provide the feed URL that is the data source for the two meetings? We would like to do some analysis on a demo server to see if we can identify what is triggering the change detection response. Thanks.

**Reply by sblack4** [reply]
Yeah, those two are pulled from https://aavirginia.org/wp-admin/admin-ajax.php?action=meetings&district=district-28-woodbridge

**Reply by sblack4** [reply]
At first I thought it might be because those two meetings have an apostrophe in the name but the aavirginia site has the same issue, here's a part of the recurring "changes" 
![image](https://github.com/user-attachments/assets/96b1e98c-0d93-4e6c-b040-0c42d3168028)

The feed for this data source is https://aablueridge.org/wp-admin/admin-ajax.php?district=district-40-skyline&district-unknown&action=meetings

**Reply by joshreisner** [reply, marked as answer]
ah that's super helpful thanks - the `entity` field that appears changed for these meetings does contain an apostrophe. we'll push an update soon that should resolve it 🤞

**Reply by sblack4** [reply]
We updated the plugin from v3.16.4 to v3.16.11 and it seems to be fixed! Thank you!

**Reply by joshreisner** [reply]
yay
